### DISCUSSION TITLE
Unable to upload the statistic to AWS metviewer dataset

### DISCUSSION THREAD

**Discussion by Ho-ChunHuang-NOAA** [dtcenter/METplus#2291] [Plot Generation] [answered]
To Whom It May Concern:

The upload was working this week but not today, please take a look and fix the problem as soon as you can.  Your drop box does not support xml file so I rename it as load_g2o_met_o3pm_aqmv7_202207_xml.txt.
```
INFO:root:METdbload Version: __version__ = "2.1.0"
INFO:root:--- *** --- Start METdbLoad --- *** ---
INFO:root:Begin time: 2023-08-05 13:21:58.538990
INFO:root:User name is: ho-chun.huang
INFO:root:Reading XML Load file
**ERROR:root:!!! XML must include user and password tags**
ERROR:root:*** <class 'NameError'> in read_xml read_db_connect ***
*** Error(s) found while reading XML file connection tag!
```
run_scrpt = /lfs/h2/emc/physics/noscrub/ho-chun.huang/METviewer_AWS/XML/load_g2o_met_o3pm_aqmv7_202207.xml
[load_g2o_met_o3pm_aqmv7_202207_xml.txt](https://github.com/dtcenter/METplus/files/12267161/load_g2o_met_o3pm_aqmv7_202207_xml.txt)

**Comment by Ho-ChunHuang-NOAA** [comment]
I need to produced some verification graphic by Monday, if someone can help on this issue as soon as they can I am greatly appreciated.

**Comment by Ho-ChunHuang-NOAA** [comment]
My test indicates that it does not recognize rds_user as valid usename <tag> and rds_pwd as valid password <tag>

**Comment by Ho-ChunHuang-NOAA** [comment]
I am taking leave staring August 7 2023.  **I will not be able to test your bug fix for this discussion.**  You do not need to notify me on the status of this discussion.

**Comment by JohnHalleyGotway** [comment]
@TatianaBurek, please take a look at this issue when you're able to. It sounds like perhaps there is an issue with user credentials. Please check to see if you're able to interact with the METviewer instance at NOAA as you'd expect.

**Comment by TatianaBurek** [comment, marked as answer]
@JohnHalleyGotway  @bikegeek @fisherhucar  I created an issue that describes a problem that causes this issue and how to fix it:
[ Password field in loading XML files can be empty #221 ](https://github.com/dtcenter/METdataio/issues/221)
I think that it should be fixed as soon as possible because this bug prevents the users to use METviewer on AWS

**Comment by BinbinZhou-NOAA** [comment]
I got  similar error when I tried to load stat data into METViewer database. Hope this can be fixed soon.

**Reply by JohnHalleyGotway** [reply]
@BinbinZhou-NOAA and @Ho-ChunHuang-NOAA, thanks for notifying of this bug. The METplus Analysis team is working on a fix and plans to include it in a METdataio-2.1.1 bugfix release in the coming weeks. I'll go ahead and mark this discussion as being "Answered". Please post any future questions or comments directly to the bugfix issue in the METdataio repository: dtcenter/METdataio#221
